inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/source2/SourceConstants.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.dataproxy.source2;
+
+public class SourceConstants {
+
+    // source host
+    public static final String SRCCXT_CONFIG_HOST = "host";
```

Comment:
I see a lot of the same constants, can we define common constants in the inlong-common module to improve code reuse?

Reply:
Well, there are indeed some places that will be repeated, but this is a temporary situation. After source2 is stabilized, the contents of the source and http directories will be replaced in parallel. The current modification idea is to modify them in parallel first, and finally remove them completely. Unused tags , the definition will not be retained later